demo-apps/cf-plugtest-client/src/main/java/org/eclipse/californium/plugtests/PlugtestClient.java

```diff
@@ -145,6 +156,9 @@ public static void main(String[] args) throws ConnectorException, IOException {
 			testCB(clientConfig.uri, context);
 			testCO(clientConfig.uri, context);
 			testCL(clientConfig.uri, context);
+
+			initOscore(clientConfig.uri, db);
```

Comment:
Please use the above cli-option also here.

Reply:
Done in commit 20aa762ebf9b41e73f67f4b9fd28afdf2f08ebd0